assets/src/domain/shared/predicates/tickets/filterTicketEntities/index.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { filter } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { isOnSale, isPending } from '../../../../../application/entities/ticket';
+import nextOnSaleOrPendingOnly from './nextOnSaleOrPendingOnly';
+import percentSoldAtOrAbove from './percentSoldAtOrAbove';
+import soldOutOnly from './soldOutOnly';
+
+interface FilterTicketEntities {
+	ticketEntities: any[];
```

Comment:
plz remove "Entity" or "Entities" from any prop names like this.
That was done in EDTRv1 because I initially wanted to differentiate between EE datetimes and JS Dates via some kind of namespace / naming convention.
My preference was to use `eventDate` or something like that to **only** mark EE Datetime objects but the other primary dev at the time came up with the idea of appending "Entity" or "Entities" to **ALL** of our object types and would not ever back down because their ideas were always superior. So to just move forwards, I acquiesced and we were stuck with having to add those suffixes to the names for ALL of our props. In these kinds of cases I think it's more sensible to add the extra namespacing to the item that is used the least in order to reduce on typing, unnecessary code, and unneeded "noise" in the code. But logic doesn't always overrule self gratification with some people. Can you tell I'm still bitter about things??? 😆 

Further to the above, although I like `eventdate` because it is clear as to what the item is (a date that is specific to an event), it causes confusion when you start trying to apply some kind of naming convention to relations involving events and datetimes (ie: `eventEventDate` ???) so I'm ok with using `Datetime` to denote an EE Datetime object and just `Date` to indicate a JS Date object, although I also often consider using `JsDate` to namespace those more clearly.

Reply:
`would not ever back down because their ideas were always superior` :man_facepalming:

Reply:
>  Can you tell I'm still bitter about things???
I think what you say makes sense. And those rules aren't very helpful, with the same success tickets would have been named `ticketItems` which is the same mouthful and useless `ticketEntities` which isn't enhancing understanding in any way.